packages/mask/src/extension/popups/pages/Wallet/TransactionDetail/types.ts

```diff
@@ -0,0 +1,4 @@
+import { type RecentTransaction, type Transaction } from '@masknet/web3-shared-base'
+import { type ChainId, type Transaction as EvmTransaction, type SchemaType } from '@masknet/web3-shared-evm'
+
+export type TransactionState = Transaction<ChainId, SchemaType> | RecentTransaction<ChainId, EvmTransaction> | undefined
```

Comment:
Move this type to `packages/mask/src/extension/popups/pages/Wallet/TransactionDetail/useTransactionLogs.ts`

Reply:
Would be used in `TransactionDetail/index.tsx` as well.